bindings/c/test/client_config_tester.cpp

```diff
@@ -92,6 +92,7 @@ class TesterOptions {
 	std::string externalClientDir;
 	bool disableLocalClient = false;
 	bool disableClientBypass = false;
+	bool tlsDisablePlainTextConnection = false;
```

Comment:
Don't need to add anything, because now you can pass any network option using the syntax --network-option-[option_name]